addons/mail/static/src/discuss/call/common/call.xml

```diff
@@ -3,10 +3,11 @@
 
     <t t-name="discuss.Call">
         <PttAdBanner/>
-        <div class="o-discuss-Call user-select-none d-flex" t-att-class="{'o-fullscreen fixed-top vw-100 vh-100': state.isFullscreen, 'o-compact': props.compact,'o-minimized': minimized, 'position-relative': !state.isFullscreen }">
-            <div class="o-discuss-Call-main d-flex flex-grow-1 flex-column align-items-center justify-content-center position-relative overflow-auto" t-on-mouseleave="onMouseleaveMain">
+        <div class="o-discuss-Call user-select-none d-flex" t-att-class="{'o-fullscreen fixed-top vw-100 vh-100': state.isFullscreen, 'o-compact': props.compact,'o-minimized': minimized and !state.isCompactCall, 'o-discuss-Collapsible-Call-area': state.isCompactCall, 'position-relative': !state.isFullscreen }">
```

Comment:
BEM convention in Discuss is as follow:
```
1. o-moduleName-componentName
2. o-moduleName-componentName-partName
3. o-modifierName
```

1. is reserved for root node
2. for all nodes except the root
3. classnames to make subtle changes to a targeted node

Examples:
```
1. o-mail-DIscuss
2. o-mail-Message-body
3. o-large (in ChatWindow, combines with .o-mail-ChatWindow.o-large)
```

So `o-discuss-Collapsible-Call-area` should probably be `o-collapsed`

_(The moduleName being `discuss` comes from desire to split `mail` and `discuss` for eventually having a `discuss` module, but this is still ongoing and long before this happens)_